src/actors/client/ActorProxyFactory.ts

```diff
@@ -1,17 +0,0 @@
-import { AbstractActor, DaprClient } from "../..";
-import Class from "../../types/Class";
-import ActorId from "../ActorId";
-import ActorProxy from "./ActorProxy";
-import ActorProxyFactoryInterface from "./ActorProxyFactory.interface";
-
-export default class ActorProxyFactory implements ActorProxyFactoryInterface {
```

Comment:
breaking change?

Reply:
There is a ProxyBuilder now that implements Proxy itself. This was old deprecated code and was not functional